packages/block-editor/src/components/global-styles/background-panel.js

```diff
@@ -379,7 +379,9 @@ function BackgroundSizeToolsPanelItem( {
 		( currentValueForToggle === 'cover' && repeatValue === undefined )
 	);
 
-	const hasValue = hasBackgroundSizeValue( style );
+	const hasValue =
+		hasBackgroundSizeValue( inheritedValue ) ||
+		hasBackgroundSizeValue( style );
```

Comment:
Note to self:

There's an potential design edge case, where theme.json specifies a `background-image` string value for `background.backgroundImage`, e.g., `url('http://test.org/image.png')`, the controls won't display a preview because the value isn't being parsed.

I think this is correct as we can't possibly parse all possible values, e.g., linear gradients, image urls and multiples of both, but maybe we could pass a dummy image to the position controls at least, e.g., "No preview available" or something, or even disable the position drag controls.

Also, the copy could be changed from "Add background image" to "Update background image" when we detect a string value for `background.backgroundImage`, e.g., 

```js
	const previewLabel =
		title ||
		( ! style?.background?.backgroundImage?.url &&
		typeof inheritedValue?.background?.backgroundImage === 'string'
			? __( 'Update background image' )
			: title );
```